### ISSUE TITLE
Update dependencies to support the latest version of React

### ISSUE THREAD

**Issue by aguidis** [esbullington/react-d3#180]
Hello,

First of all, thanks for this module.

I installed react-d3 module in my app but I encountered this error when I tried to rendering the chart :
```
Uncaught TypeError: Can't add property context, object is not extensible
```
This error is due to the use of different versions of React (0.12 for react-d3 and 0.13 for my app).

@JustinCarmony explained it very well in his recent post : http://www.justincarmony.com/blog/2015/04/02/webpack-react-multiple-versions-issues/
He succeeded in resolving this issue by using the webpack aliases.

The fact is I use Browserify and not Webpack. Browserify don't give us the ability to create alias natively. I just found this module https://www.npmjs.com/package/aliasify which seems to do the same work but I didn't tested yet.

If you could included the last version of React in the next release it would be nice.

Thanks,

Adrien

**Comment by esbullington** [comment]
Thanks for letting me know.  I've just released a new patch 0.3.1 that hopefully will take care of the issue.  I didn't want to exclude people still on React 0.12 from using the new version of the library, and still let people on 0.13 use it, so I went with `>0.12`.  Let me know if that doesn't work for you.

**Comment by aguidis** [comment]
Thanks for your answer and for the patch :) After few hours of tests, I finally decided to use Webpack so I can force D3 to use my react version and it works.

But the patch is a great news nevertheless !
